docs/user_guides/single_table/constraints.rst

```diff
@@ -161,38 +161,60 @@ The ``GreaterThan`` constraint can also be used to guarantee a column is greater
 than a scalar value or specific datetime value instead of another column. To use
 this functionality, we can pass:
 
--  the scalar value for ``low``
--  the scalar value for ``high``
--  a boolean indicating ``low`` or ``high`` is a scalar
+-  the scalar value for ``low`` or the scalar value for ``high``
+-  a flag indicating whether ``low`` or ``high`` is a scalar
 
 .. ipython:: python
     :okwarning:
 
     salary_gt_30000_constraint = GreaterThan(
         low=30000,
         high='salary',
+        scalar='low',
         handling_strategy='reject_sampling'
     )
 
+Optionally, when constructing ``GreaterThan`` constraint we can specify 
+more than a single column in either the ``high`` or ``low`` arguments. 
+For example, we can create a ``GreaterThan`` constraint that that ensures 
+that both the years in the company and prior years of experience is more 
+than one year.
+
+.. ipython:: python
+    :okwarning:
+
+    experience_years_gt_one_constraint = GreaterThan(
+        low=1,
+        high=['years_in_the_company', 'prior_years_experience'],
+        scalar='low',
+        handling_strategy='reject_sampling'
+    )
+
+.. note::
```

Comment:
Minor adjustment 

```
Warning! Passing a list of columns to the ``high`` or ``low`` arguments is only possible when
the other one has been passed as a scalar or one of them is a single column! In other 
words, passing a more that one column in ``high`` AND more than one column in ``low``
is not possible.
```

Reply:
Yeah, it needs to be adjusted wrt the possibility of passing a single column. I would try to phrase it using positive statements rather than negative, though. Here's another proposal, adding the single column to the previous one:

```
Warning! Passing a list of columns to the `high` or `low` arguments is only possible when
the other one has been passed as a single column name or scalar value! If you need to
compare multiple ` high`` columns against multiple ``low`` columns (or vice versa), you
need to decompose one of the ends, ``high`` or ``low``, into multiple single column
names and define one ``GreaterThan`` constraint for each one of them.
```